dotnet/samples/Hello/Hello.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
SI there anything here that can go into the top level AutoGen.sln? other folks told me e don't want multiple solution files... I'm not sure why?